src/Morphir/IR/SDK/Key.elm

```diff
@@ -0,0 +1,486 @@
+{-
+   Copyright 2020 Morgan Stanley
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-}
+
+
+module Morphir.IR.SDK.Key exposing (..)
+
+import Dict exposing (Dict)
+import Morphir.IR.Documented exposing (Documented)
+import Morphir.IR.Module as Module exposing (ModuleName)
+import Morphir.IR.Name as Name exposing (Name)
+import Morphir.IR.Path as Path exposing (Path)
+import Morphir.IR.SDK.Common exposing (tFun, tVar, toFQName, vSpec)
+import Morphir.IR.Type exposing (Specification(..), Type(..))
+
+
+moduleName : ModuleName
+moduleName =
+    Path.fromString "Key"
+
+
+moduleSpec : Module.Specification ()
+moduleSpec =
+    { types =
+        Dict.fromList
+            [ ( Name.fromString "Key0", OpaqueTypeSpecification [] |> Documented "" )
+            , ( Name.fromString "Key2", OpaqueTypeSpecification [] |> Documented "" )
+            , ( Name.fromString "Key3", OpaqueTypeSpecification [] |> Documented "" )
+            , ( Name.fromString "Key4", OpaqueTypeSpecification [] |> Documented "" )
+            , ( Name.fromString "Key5", OpaqueTypeSpecification [] |> Documented "" )
+            , ( Name.fromString "Key6", OpaqueTypeSpecification [] |> Documented "" )
+            , ( Name.fromString "Key7", OpaqueTypeSpecification [] |> Documented "" )
+            , ( Name.fromString "Key8", OpaqueTypeSpecification [] |> Documented "" )
+            , ( Name.fromString "Key9", OpaqueTypeSpecification [] |> Documented "" )
+            , ( Name.fromString "Key10", OpaqueTypeSpecification [] |> Documented "" )
+            , ( Name.fromString "Key11", OpaqueTypeSpecification [] |> Documented "" )
+            , ( Name.fromString "Key12", OpaqueTypeSpecification [] |> Documented "" )
+            , ( Name.fromString "Key13", OpaqueTypeSpecification [] |> Documented "" )
+            , ( Name.fromString "Key14", OpaqueTypeSpecification [] |> Documented "" )
+            , ( Name.fromString "Key15", OpaqueTypeSpecification [] |> Documented "" )
+            , ( Name.fromString "Key16", OpaqueTypeSpecification [] |> Documented "" )
+            ]
```

Comment:
Did you use automation to generate any of this? If so can you include the code in the comments so that we can make changes later without having to do it manually?